kwikapi/exception.py

```diff
@@ -3,7 +3,11 @@
 
 
 class ResponseError(Exception):
-    pass
+    def __init__(self, response):
+        super(ResponseError, self).__init__(response)
+        self.message = response['message']
```

Comment:
@prashanthellina 
user(developer) can get `message`, `code` from ResponseError object if any callee service fails.

with reference to [example](https://github.com/deep-compute/kwikapi/#examples) in README, 
```python
# client_add.py
from kwikapi import Client

c = Client('http://www.example.com:8888/api/', version='v1', protocol='json', raise_exception=False)

res = c.addd(a=10, b=10) # returns exception object
if isinstance(res, Exception):
    print('Error mesage: {}'.format(res.message))
else:
    print(res)
```
Output:
```bash
$ python client_add.py
   mesage: Unknown API Function: "addd"
```